src/exercises/Exercises.js

```diff
@@ -251,20 +253,15 @@ export default function Exercises({
     setShowFeedbackButtons(!showFeedbackButtons);
   }
 
-  let wordSourceText = articleInfo ? (
-    <>"{articleInfo.title}"</>
-  ) : (
-    <>{strings.wordSourceDefaultText}</>
-  );
-
   const CurrentExercise = exerciseSession[currentIndex].type;
   return (
     <s.ExercisesColumn>
-      <s.LittleMessageAbove>
-        {strings.wordSourcePrefix} {wordSourceText}
-      </s.LittleMessageAbove>
+      {!articleID ? (
+        <s.LittleMessageAbove>
+          {strings.wordSourcePrefix} {strings.wordSourceDefaultText}
```

Comment:
Emma, do you think we can unify things more between the extension and the web app if we remove the "Exercises" title, and we put the link to the article as in the example? Or maybe below? I can commit this code if you think it makes sense.

<img width="957" alt="image" src="https://user-images.githubusercontent.com/464519/159792497-6ba437bd-3419-4041-91ee-c106215a53c3.png">

<img width="943" alt="image" src="https://user-images.githubusercontent.com/464519/159792408-091f256e-21a7-415e-9a6a-6f7f1f5ce1f5.png">

Reply:
I understand why this is a good implementation in the web app, but I'm still not sure it's the best way to go for the extension. I think that "Back to article" is an important functionality in the extension - and it should be very clear to the user that they can go back. In the web app, you can just navigate via the sidebar if you don't want to do the exercises - but you can't in the extension. If you go into exercises and regret halfway - or just want to see what exercises are, then the only other option in the extension is to close the window. And we don't want people to shut down the extension, because they can't go back from exercises. So, i don't feel like this implementation of "Go to article" is obvious enough for the users. 

Frida suggested that we could write something like "Go back to reading ..." instead of "Exercises based on ...". Maybe that could work? Either in both the extension and the web app - or we could use a prop to determine the text.

Reply:
My reasoning is that if you don't want to do the exercises, then it's also very unlikely that you are going to go back to the article. You just read it... Given that it's a very improbable, and surely not frequent situation, I think it would be granted that in the rare case where this happens, you could close the extension and then re-open it again. 

I guess we can do a "go back to reading" for now. 
And then we collect info about how people use this, and then we improve it later based on their behavior?

Reply:
Sure, we can do that. So, we go with your implementation - with the link above the exercises - but write "Go back to reading: Stahl- und Alumomo..." instead?

Reply:
Let's try that!

Reply:
Ping me when this is done so I can merge :)

Reply:
Oh, I thought that you had the code for this change. I'll look at it then - but i'm still not sure what to do with the buttons in the conversation above.

Reply:
I made these changes on my branch now - I have not pushed it though. Still not sure about the backToArticle and backToReading - since we need both functionalities.

Reply:
Ah. You're referring to this?

> Oh wait. We can't do that. BackToArticle is only possible when it is the exercises for a specific article. So "backToReading" is needed when they enter through zeeguu.org/exercises.

Can't it be solved if the callback is called backToReading and we pass different behavior based on the context in which we initialize the Exercises?

Reply:
Yeah - I think that is what I did now. I just requested a new review :)